documentation/source/NuRadioReco/pages/times.rst

```diff
@@ -1,52 +1,95 @@
+Work similar to `trace_start_time <NuRadioReco.framework.base_trace.BaseTrace.get_trace_start_time>` in channels, but with a caveat: Since EM-waves move, electric_field objects hold a position (relative to the station they are associated with). The E-field is therefore defined as the field an observer would measure at the given position. Note that this position does not necessarily have to coincide with the position of a channel the E-field is associated with. This is the case for (some) cosmic-ray simulations where the same E-field at the surface is used for all surface LPDAs.
+
+Trigger times
+^^^^^^^^^^^^^
+The `trigger_time <NuRadioReco.framework.trigger.Trigger.get_trigger_time>`,
+which is the time at which the trigger fired, is stored in the `Trigger <NuRadioReco.framework.trigger.Trigger>`
+object (which can be obtained using `station.get_trigger() <NuRadioReco.framework.station.Station.get_trigger>`).
+This is the time at which the trigger condition was first fulfilled.
+As for the trace_start_time, the trigger time is defined relative to the
+`station_time <NuRadioReco.framework.station.Station.get_station_time>` .
+
+
+Overview of modules that affect time
+------------------------------------
+We list all relevant modules that are used for a MC simulation and reconstruction. For a pure data reconstruction, the first two modules are not used.
 
-overview of modules that alter time
-===================================
-We list all relevant modules that is used for a MC simulation and reconstruction. For a pure data reconstruction, the first few modules are just not used
+* `NuRadioReco.modules.io.coreas`: CoREAS reader prepends n samples to the simulated trace. This is done so that the trace does not directly start with the pulse and to have a good frequency resolution.
 
-* readCoREAS: CoREAS reader prepends n samples to the simulated trace. This is done so that the trace does not directly start with the pulse and to have a good frequency resolution.
+* `NuRadioReco.modules.efieldToVoltageConverter`: the voltage traces are delayed compared to the electric field signal due to the geometric separation of the antennas and cable delays. This is accounted for by shifting the `trace_start_time <NuRadioReco.framework.base_trace.BaseTrace.get_trace_start_time>`.
 
-* efieldToVoltageConverter: the traces are rolled (rotated) according to the time delay due to the geometric separation of the antennas and cable delays.
+* `NuRadioReco.modules.RNO_G.hardwareResponseIncorporator`, `NuRadioReco.modules.ARIANNA.hardwareResponseIncorporator`, `NuRadioReco.modules.ARA.hardwareResponseIncorporator`:
+
+  If ``sim to data=True``:
 
-* hardwareResponseIncorporator (sim to data):
     * the channel traces are folded with the amplifier response which also includes some time delay
```

Comment:
I would add a statement that this time delay is reflected by moving the pulse within the trace and not by adjusting the trace_start_time